RecoTracker/DeDx/python/dedxEstimators_cff.py

```diff
@@ -78,3 +92,9 @@
 
 doAlldEdXEstimatorsTask = cms.Task(dedxTruncated40 , dedxHarmonic2 , dedxPixelHarmonic2 , dedxPixelAndStripHarmonic2T085 , dedxHitInfo)
 doAlldEdXEstimators = cms.Sequence(doAlldEdXEstimatorsTask)
+
+doFastdEdXEstimatorsTask = cms.Task(dedxHarmonic2, dedxPixelHarmonic2)
+doFastdEdXEstimators = cms.Sequence(doFastdEdXEstimatorsTask)
+
+fastSim.toReplaceWith(doAlldEdXEstimatorsTask, doFastdEdXEstimatorsTask)
+fastSim.toReplaceWith(doAlldEdXEstimators, doFastdEdXEstimators)
```

Comment:
To me `doFastdEdXEstimators` Sequence looks redundant as the necessary replacement is done already with `doFastdEdXEstimatorsTask` (which could be further simplified to
```python
fastSim.toReplaceWith(doAlldEdXEstimatorsTask, cms.Task(dedxHarmonic2, dedxPixelHarmonic2))
```
)